dash/__init__.py

```diff
@@ -19,7 +19,7 @@
 from . import html  # noqa: F401,E402
 from . import dash_table  # noqa: F401,E402
 from .version import __version__  # noqa: F401,E402
-from ._callback_context import callback_context  # noqa: F401,E402
+from ._callback_context import callback_context, ctx  # noqa: F401,E402
```

Comment:
Does this need to be a separate instance? Can we just say `ctx = callback_context` after all the imports, and remove the [extra instantiation](https://github.com/plotly/dash/pull/1952/files#diff-f19c52717a102d5ef8b91b157d234b9075d07bbcc2cd1d0a0cd8060bc9c7f608R290)?